mac.sh

```diff
@@ -152,16 +151,32 @@ install_bundler_1_17_3() {
   bundle config --global jobs $((number_of_cores - 1))
 }
 
-install_postgresql() {
-  brew_install_or_upgrade 'postgresql@9.6'
-  brew link postgresql@9.6 --force
-  brew_launchctl_restart 'postgresql'
-  if [ `psql -U postgres -c "select 1" &> /dev/null` ]; then
-    /usr/local/bin/createuser -U `whoami` --superuser postgres
+ # PostgreSQL 9.6 hit EoL and is no longer supported by brew. This step now downloads the PG App - https://postgresapp.com/ - to install 9.6
+ # You will need to manually start the Database however. See the StartingPostgresql.md guide in this repo.
+ install_postgresql() {
+  # brew_install_or_upgrade 'postgresql@9.6'
+  # brew link postgresql@9.6 --force
+  # brew_launchctl_restart 'postgresql'
+  # if [ `psql -U postgres -c "select 1" &> /dev/null` ]; then
+  #   /usr/local/bin/createuser -U `whoami` --superuser postgres
+  # fi
+
+  if [ -z `psql --version`]; then
```

Comment:
i wanted to try this part of the script out but I'm on linux and can not! I read a bit about `hdiutil` which I've never seen before but all seems perfect to me